requirements/requirements_dev.in

```diff
@@ -1,10 +1,12 @@
--r requirements.txt
--c requirements.txt
+-r requirements.in
+-c requirements.in
```

Comment:
Why this?
We should restrict our development environment's requirement (requirements_dev) to exactly the same requirements as the production environment's ones (requirements.txt) and not to their open counterpart (requirements.in).